### DISCUSSION TITLE
[Question]: Front page's cameras live view playing not capped?

### DISCUSSION THREAD

**Discussion by bakedhorse** [blakeblackshear/frigate#11937] [Ask A Question] [answered]
### What is your question:

So I love the new feature of the front page cameras being able to play live when there's motion, very cool feature.

However is this capped in any way? I'm just curious as I have 12 cameras and I don't want to smack my internet or phone provider with a bandwidth of 12 4K cameras at once (if such scenario does occur).
I know you can disable this in the General Settings, but it'd be nice if maybe a custom cap could be provided for anyone wanting to use it but don't want to destroy their bandwidth.

**Comment by NickM-27** [comment, marked as answer]
only cameras that are visible on the screen will switch to live. So for example on mobile that would only be 2-3 cameras at a time. 

In general, it is recommended to use a mid stream for live view instead of the main stream.

**Reply by bakedhorse** [reply]
Unfortunately I can't use a mid stream unless I wanted to transcode every camera. 2-3 cameras at a time is not bad for mobile, so that's nice. But on PC I can see 9 cameras at once and I know I'll still use my laptop on the go. 
Sadly, we get some very windy days were I live and there's not really much I can do about the motion detection unless I want to miss objects/motion detections.

**Reply by NickM-27** [reply]
in a future version we will likely make it so the grid shows sub streams and the main live view shows the main stream
